base/applications/mplay32/lang/it-IT.rc

```diff
@@ -11,21 +11,21 @@ BEGIN
     END
     POPUP "&Play"
     BEGIN
-        MENUITEM "Play/&Pause\tCtrl+P", IDC_PLAY
+        MENUITEM "Play/&Pausa\tCtrl+P", IDC_PLAY
```

Comment:
And "Play" is Italian?

Reply:
@HBelusca Italian borrows a considerable number of some English words. "Play" does have its own translation within Italian of course, which is *Giocare*, but it is used for a different context therefore it does not make sense to have "Play" translated as it would confuse Italian speakers.

Reply:
Ok.